src/mdlComponentHandler.js

```diff
@@ -51,6 +51,18 @@ var componentHandler = (function() {
   }
 
   /**
+   * Returns an array of the classNames of the upgraded classes on the element.
+   * @param {HTMLElement} element The element to fetch data from.
+   * @return {[string]}
+   * @private
+   */
+  function getUpgradedListOfElement_(element) {
```

Comment:
Added `getUpgradedListOfElement_(element)` for getting a list of names of the upgraded classes.